<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_superior">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>811</width>
       <height>66</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>676</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_minimizar">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/frangarayco/Pictures/guion.jpg</normaloff>C:/Users/frangarayco/Pictures/guion.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>38</width>
          <height>38</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_cerrar">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/frangarayco/Pictures/cerrarPiola.jpg</normaloff>C:/Users/frangarayco/Pictures/cerrarPiola.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>38</width>
          <height>38</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_inferior">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>59</y>
       <width>811</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(61,61,61);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(255,255,255);
font: 77 10pt &quot;Arial Black&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>5</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>INICIO DE SESION: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Nombre de usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Contrase√±a:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_user">
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_contra">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mostrar">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>76</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_iniciar_sesion">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>170</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Iniciar Sesion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Si no tenes cuenta, create una aca </string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_registro">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);</string>
      </property>
      <property name="text">
       <string>Registrarme</string>
      </property>
     </widget>
     <widget class="QLabel" name="signal_login">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>500</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,206,151);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
